### ISSUE TITLE
Clarification on data used for pretraining

### ISSUE THREAD

**Issue by newalexander** [RishikeshMagar/Crystal-Twins#3]
Hello, and thanks for making this work accessible! I am curious if you can give a bit more detail on the matminer and hMOF datasets you used for your pretraining.

For the data from Matminer, which datasets did you use? If I download everything from Matminer's datasets (https://hackingmaterials.lbl.gov/matminer/dataset_summary.html) and filter for those containing material structure information, I get 614,427 rows, which doesn't match the 152,368 reported in the paper. Does removing duplicates reduce this down to 152,368, or did you only work with a subset of those datasets?

For the data from hMOF, downloading the full hMOF zip file from here (https://mof.tech.northwestern.edu/databases) yields 137,953 CIFs, which doesn't match the 275,907 reported in the paper. Did you include other databases from the [MOFXDB](https://mof.tech.northwestern.edu/)?

**Comment by RishikeshMagar** [comment]
Hi there,
Thanks for your interest in our work. 

 "I get 614,427 rows, which doesn't match the 152,368 reported in the paper."
 For pretraining since you don't need the labels we just did unique on the whole matminer dataset. This was done around Aug 2021, so the matminer size may have been lower. However, if you get a larger unique set, I expect the performance to improve.

"hMOF yields 137,953 doesn't match 275,907"
Now that is oversight from our end, if you notice it is exactly twice. We had MOF data separated by CO2 and CH4 adsorption  and although the MOFs were the same for the both we did not do unique here. Ideally, it should have been unique here but since MOFs have a larger number of atoms and the augmentations we introduced may have ensured that the there is still a sense of difference between MOFs. Moreover, having MOFs or larger crystalline systems is important for the pretraining stage. We were unable to train ALIGNN on MOFs because of memory requirements but we did train it on smaller dataset it did not show performance gains.
